digiwf-connector/digiwf-camunda-rest-connector/src/main/java/de/muenchen/oss/digiwf/connector/adapter/camunda/rest/DigiWFCamundaConnectorAutoConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package de.muenchen.oss.digiwf.connector.adapter.camunda.rest;
```

Comment:
Die Autoconfiguration würde ich aus dem adapter package rausziehen. Das package wäre dann `de.muenchen.oss.digiwf.connector`

Reply:
Die gilt aber nur für das Package de.muenchen.oss.digiwf.connector.adapter.camunda.rest

Reply:
Ja, genau 👍🏻

Reply:
Warum soll die Autoconfig dann nicht in dem base package liegen, ab dem sie gilt?

Reply:
Wenn man mehrere Adapter erstellt, dann könnte man über die Autoconfig auswählen welche Beans erstellt werden. z.B. wenn wir einen Camunda Rest und einen Camunda Streaming Adapter hätten.

Reply:
Mich hat das iritiert, dass die Config und Properties direkt neben dem Code liegen. Bisher hatten wir den Fall noch nicht, dass Code in einem Starter ist. Meistens liegen die Configs sogar in einem eigenen configuration package.

Mach's am besten so wie es für dich am schlüssigsten ist